<a href="https://colab.research.google.com/github/jherrerae/Project2/blob/main/Project_2_Week_1_v2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [ ]:
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import numpy as np

In [ ]:
filename="/content/drive/MyDrive/Coding Dojo/Classes/Datasets/E-Commerce Shipping Data_Train.csv"
df=pd.read_csv(filename)
df.head(5)

ID Warehouse_block Mode_of_Shipment  Customer_care_calls  Customer_rating  \
0   1               D           Flight                    4                2   
1   2               F           Flight                    4                5   
2   3               A           Flight                    2                2   
3   4               B           Flight                    3                3   
4   5               C           Flight                    2                2   

   Cost_of_the_Product  Prior_purchases Product_importance Gender  \
0                  177                3                low      F   
1                  216                2                low      M   
2                  183                4                low      M   
3                  176                4             medium      M   
4                  184                3             medium      F   

   Discount_offered  Weight_in_gms  Reached.on.Time_Y.N  
0                44           1233                    1  
1                59           3088                    1  
2                48           3374                    1  
3                10           1177                    1  
4                46           2484                    1

In [ ]:
df.shape

(10999, 12)

First choice: dataset 1

1) Source of data

Kaggle
https://www.kaggle.com/datasets/prachi13/customer-analytics

2) Brief description of data

An international e-commerce company based wants to discover key insights from their customer database.

3) What is the target?

Reached on time: It is the target variable, where 1 Indicates that the product has NOT reached on time and 0 indicates it has reached on time.


4) Is this a classification or regression problem?

Classification

5) How many features?

12 columns

6) How many rows of data.

10999

7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?

combination of categorical and numerical values, evaluate if variables hshould be taken into account in the model